bin.js

```diff
@@ -45,13 +50,21 @@ function getConfig(file) {
     // point to the particular config
     configFile: file,
   })
-  var config = cliEngine.getConfigForFile(file)
-  return config
+  return cliEngine.getConfigForFile(file)
```

Comment:
Would you mind changing this to `return cliEngine.getConfigForFile()` (without arguments, as you pointed out it'll work this way).